sharepoint/sharepoint-ps/sharepoint-online/Stop-SPOTenantRename

```diff
@@ -0,0 +1,141 @@
+Skip to content
+Search or jump to…
+Pull requests
+Issues
+Marketplace
+Explore
+ 
+@WayneEwington 
+WayneEwington
+/
+office-docs-powershell
+Public
+forked from MicrosoftDocs/office-docs-powershell
+1
+0
+883
+Code
+Pull requests
+1
+Actions
+Projects
+Wiki
+Security
+Insights
+Settings
+office-docs-powershell/sharepoint/sharepoint-ps/sharepoint-online/Stop-SPOTenantRename
+@WayneEwington
+WayneEwington Create Stop-SPOTenantRename
+…
+Latest commit 638e164 1 minute ago
+ History
+ 1 contributor
+95 lines (65 sloc)  2.68 KB
+  
+---
+external help file: sharepointonline.xml
+Module Name: Microsoft.Online.SharePoint.PowerShell
+online version: https://docs.microsoft.com/powershell/module/sharepoint-online/stop-spotenantrename
+applicable: SharePoint Online
+title: Stop-SPOTenantRename
+schema: 2.0.0
+author: WayneEwington
+ms.author: waynewin
+ms.reviewer:
+---
+
+# Stop-SPOTenantRename
+
+## SYNOPSIS
+
+ [!IMPORTANT]
```

Comment:
```suggestion
 [!IMPORTANT]
```

Same as https://github.com/MicrosoftDocs/OfficeDocs-SharePoint-PowerShell/pull/11 and https://github.com/MicrosoftDocs/OfficeDocs-SharePoint-PowerShell/pull/12.